titan-web-client/src/modules/roster/branches/index.js

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { withRouter } from 'react-router-dom';
+import PageHeader from '../../../layouts/dashboard/components/PageHeader/PageHeader';
+import PageHeaderTitle from '../../../layouts/dashboard/components/PageHeader/PageHeaderTitle';
+import { ContentBlock } from '../../../components/block/ContentBlock';
+import BranchCardGrid from '../components/BranchCardGrid';
+import OrganizationsService from '../../../http/OrganizationsService';
+
+class BranchesScene extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.organizationHttpService = new OrganizationsService();
+    this.state = {
+      branches: [
```

Comment:
Is this left over debug/playground code? These fields should be provided from the server. Setting `branches` as an empty array should be sufficient. We can add a loading spinner at a later time if necessary.